resources/assets/components/Tags/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { map } from 'lodash';
+
+class Tags extends React.Component {
+  render() {
+
+    return (
+        <ul className="form-actions -inline">
+          <li><b> Tags </b></li>
```

Comment:
This is a heading so should be pulled out from the list and given a heading tag:

```jsx
<h4>Tags</h4>
<ul className="form-actions -inline">
  {/* ... */}
```